moveit_core/kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -598,6 +600,36 @@ class KinematicsBase
   std::map<int, double> redundant_joint_discretization_;
   std::vector<DiscretizationMethod> supported_methods_;
 
+  /**
+   * @brief Enables kinematics plugins access to parameters that are defined
+   * for the 'robot_description_kinematics' namespace.
```

Comment:
Could you add a comment explaining how it tried to find the param in the root namespace first, then in the group name subgroup second?

Reply:
https://github.com/ros-planning/moveit/pull/701/commits/45a35c83951d0bab04db7318554791461d841f1c adds some explanation for the lookup order.